mllib/src/main/scala/org/apache/spark/ml/regression/LinearRegression.scala

```diff
@@ -209,7 +219,13 @@ class LinearRegression(override val uid: String)
 
     if (handlePersistence) instances.unpersist()
 
-    copyValues(new LinearRegressionModel(uid, weights, intercept))
+    val model = copyValues(new LinearRegressionModel(uid, weights, intercept))
```

Comment:
Actually, I need to call copyValues there because `model.transform` used on L223 requires `predictCol` to be set. `JavaLinearRegressionSuite#linearRegressionWithSetters()` will fail otherwise.